include/led-matrix.h

```diff
@@ -84,6 +84,9 @@ class RGBMatrix : public Canvas {
   void set_luminance_correct(bool on);
   bool luminance_correct() const;
 
+  void SetBrightness(uint8_t brightness);
```

Comment:
The problem here is, that if you draw something and then change the brightness, only new drawings will get that new brightness. That's why I was not sure if implementing the function in FrameCanvas is such a good idea (also brightness should be a global setting to the device like luminance correction) and not apply to a single frame.